.gitignore

```diff
@@ -1,5 +1,6 @@
 *TAGS
 _build
+_opam
```

Comment:
can it appear in nested folders? if not, `/_opam` will make it only apply at the root, like with `/_obuild` (which i'm gonna go delete, it was for ocp-build). we probably could change to `/_build` too

Reply:
I think it can't

Reply:
_build can appear in nested folders tho